server/index.js

```diff
@@ -16,13 +16,13 @@ app.listen(PORT, () => {
 
 // get request example
 app.get('/movies', (req, res) => {
-  console.log(req.body);
-  let movieId = req.body.id;
+  console.log(req.query);
+  let movieId = req.query.movieID;
   db.getMovieInfo(movieId, (err, results) => {
     if (err) {
       res.sendStatus(500);
     } else {
-      res.json(results);
+      res.json(results[0]);
```

Comment:
just curious about what prompted you to specify the first index. From what I can see, the movieId is coming from what you hard coded into the state on line 9 of client/components/App.jsx and what I predict would be returned  (and I could totally be wrong ) would look like this with the values for the movie with movieId: 1 filled in:
 {name: String,
  info: {
    description: String,
    release: String,
    genre: String,
    score: Number,
    runtime: Number,
    image: String,
    rating: String
  }
I'd love to know what I might be missing here.